code/script.js

```diff
@@ -1,53 +1,292 @@
+          <option value="choose" selected disabled>--Please select--</option>
+          <option value="Dachshund">Dachshund</option>
+          <option value="Golden retriever">Golden retriever</option>
+          <option value="Poodle">Poodle</option>
+          <option value="Pomeranian">Pomeranian</option>
+          <option value="Great Dane">Great Dane</option>
+        </select> 
+        <button class="send-btn" id="send-btn" type="submit">Confirm</button>
+      </form>`;
+      break;
+    case "horse":
+      inputWrapper.innerHTML = `
+      <form id="animal-type">
+        <select class="curtain" id="horse-select">
+          <option value="choose" selected disabled>--Please select--</option>
+          <option value="Shetland pony">Shetland pony</option>
+          <option value="Thoroughbred">Thoroughbred</option>
+          <option value="Andalusian">Andalusian</option>
+          <option value="Connemara">Connemara</option>
+          <option value="Ardennais">Ardennais</option>
+        </select> 
+        <button class="send-btn" id="send-btn" type="submit">Confirm</button>
+      </form>`;
+      break;
+  }
+};
+
+// This function inserts the gender option buttons into the html
+const addGenderButtons = () => {
+  inputWrapper.innerHTML = `
+    <button class="send-btn" id="male">Male</button>
+    <button class="send-btn" id="female">Female</button>`;
+};
+
+// This function inserts a required contact form into the html
+const addContactForm = () => {
+  inputWrapper.innerHTML = `
+  <form id="contact-form">
+    <label>
+      <input type="tel" name="tel" id="tel" placeholder="Phone" required/>
+    </label>
+    <label>
+      <input type="email" name="email" id="email" placeholder="E-mail" required/>
+    </label>
+    <button class="send-btn" id="send-btn" type="submit">Send</button>
+  </form>`;
+};
+
+// This function gives us the name of the user
+const handleNameInput = (event) => {
+  event.preventDefault();
+  userName = nameInput.value;
+  nameInput.value = "";
+  // Check if the user has entered a name before moving on
+  if (userName) {
+    showMessage(userName, "user");
+    setTimeout(chooseAnimal, 1000);
+  } else {
+    showMessage("I'm not sure that's a name. Please try again.", "bot");
+  }
+};
+
+// The user is asked to choose between different animals by clicking buttons
+const chooseAnimal = () => {
+  showMessage(
+    `Happy to meet you, ${userName}! What type of animal are you interested in adopting?`,
+    "bot"
+  );
+  // Animal choice buttons are added to the html
+  addAnimalButtons();
+  // Depending on which button is clicked, the id of the clicked button is added to the animalChoice variable
+  const animalButton = document.querySelectorAll(".send-btn");
+  console.log(animalButton);
+  animalButton.forEach((animalButton) => {
+    animalButton.addEventListener("click", (event) => {
+      animalChoice = event.target.id;
+      console.log(animalChoice);
```

Comment:
Remember to remove console.logs, they should only be used during development and not in production.